components/core/src/crypto/keys/mod.rs

```diff
@@ -275,6 +286,17 @@ where
                 continue;
             }
         };
+
+        let file = File::open(p.path())?;
```

Comment:
This change is causing issues with system metadata files in the directory. For example in OS X, if there is a `.DS_Store` file, it will cause the following panic.
```
✗✗✗
✗✗✗ stream did not contain valid UTF-8
✗✗✗
```

This seems to result from opening all files in the directory and attempting to decode them blindly.